tests/test_scalyr.py

```diff
@@ -915,7 +915,6 @@ def test_parse_scalyr_sampling_rules(monkeypatch, scalyr_env, fx_scalyr):
         'scalyr_sampling_rules': SCALYR_SAMPLING_RULES,
     })
 
-    print(agent.scalyr_sampling_rules)
```

Comment:
Why print within a test case?

Reply:
That's why I'm removing it. Accidentally found this it the code

Reply:
I obviously can't tell the difference between red and green and need to practice reading diffs more. 🤦

Reply:
Test your color vision maybe 😁

![image](https://user-images.githubusercontent.com/556933/82053988-86ef3e00-96be-11ea-8e1b-465464c5fa06.png)